code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
That makes sense, but the error I'm continuing to get implies to me that I'll be unable to keep this as a define. I think it's checking the list even though one condition of the && fails before it, this throwing a runtime.